app.js

```diff
@@ -1,11 +1,23 @@
 const express = require("express");
 const app = express();
 const endpointsJson = require("./endpoints.json");
+const getTopics = require("./controllers/topics.controllers");
 
 app.use(express.json());
```

Comment:
We don't need this line of code yet - we should only have code in our files that we actually use 👀